jep/400/README.adoc

```diff
@@ -0,0 +1,224 @@
+
+When users create or import new applications, the build packs get applied to generate the default _Dockerfile_ for building docker images, the _Jenkinsfile_ for the CI / CD together with the Helm chart files to deploy it on Kubernetes.
+
+Teams and communities can share and customise their build packs allowing builds, CI and CD to be automated for a wide range of languages and frameworks.
+
+=== Helm Charts
+
+Helm charts are the standard packaging mechanism for installing and upgrading applications on Kubernetes.  Jenkins X provides a Helm chart so it is easy to install Jenkins X on any kubernetes cluster. 
+
+Helm charts can also be composed and configured in various ways; so we can provide customised Jenkins X helm charts tailored to different public clouds so that Jenkins X can automatically make the best use of  the available services on the cloud without the user needing to configure anything. E.g. so that Jenkins X can use Stackdriver on Google or CloudWatch on AWS for application and pipeline logs.
+
+Traditionally the plugin model of Jenkins has been to add jars inside the JVM of the Jenkins master. With Kubernetes the extension model can be more flexible and allow extension through separate microservices and configurations distributed in Helm charts.  These microservices then communicate with each other via Kubernetes resources which are exposed in the Kubernetes REST APIs, kubernetes language clients, Kubernetes Dashboard and the kubectl command line tool.
+
+One goal of Jenkins X is to foster a community of Jenkins X addons which are optional Helm charts that can extend Jenkins X with additional capabilities; whether its for different source control repositories, issue management, code quality tools, testing tools, chat integration, UIs or operational management tooling.
+
+E.g. we expect there to be Jenkins X addons like:
+
+* Nexus / Artifactory for Java artifacts
+* Chart Museum/Monocular for Helm chart repositories
+* Metrics and monitoring (e.g. Grafana and Prometheus)
+* Gitea for on premise git hosting
+
+=== Kubernetes Native Jenkins
+
+Jenkins X is not a general purpose Jenkins that can modified to anything. Instead, it is tailored to focus on Kubernetes and Cloud Native use cases, hence the name “Kubernetes native Jenkins.” 
+
+Jenkins X includes Jenkins core with a specific set of plugins bundled together to provide the necessary user experience out of the box, without the user needing to “assemble their own LEGO blocks.”
+
+Over time, we see an opportunity to improve Jenkins core itself based on our learning in Jenkins X, so that Jenkins itself can be used in more cloud native configurations. This should benefit not only Jenkins X but other uses of Jenkins. E.g. to allow storage of Jenkins resources (config, pipelines, builds, credentials, artifacts, logs) outside of the Jenkins master local file system to avoid a single point of failure and make managing persistent disk easier.
```

Comment:
Should emphasize that changes like these would need to be upstream JEPs.

BTW please follow the practice of using a separate (AsciiDoc/Markdown) source line for each distinct sentence or major phrase, so that reviewers can target their comments more precisely.